libraries/AP_RCProtocol/AP_RCProtocol_CRSF.cpp

```diff
@@ -337,6 +340,28 @@ bool AP_RCProtocol_CRSF::process_telemetry(bool check_constraint)
     return true;
 }
 
+// process link statistics to get RSSI
+void AP_RCProtocol_CRSF::process_link_stats_frame(void* data)
+{
+    LinkStatisticsFrame* link = (LinkStatisticsFrame*)data;
+
+    uint8_t rssi_dbm;
+    if (!link->active_antenna) {
```

Comment:
Any reason to invert here?

Reply:
I've made it ==0 - the antennas are 0 and 1